addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1266,6 +1270,86 @@ options.registry.parallax = options.Class.extend({
     onMove: function () {
         this._refreshPublicWidgets();
     },
+
+    //--------------------------------------------------------------------------
+    // Options
+    //--------------------------------------------------------------------------
+
+    /**
+     * Build/remove parallax.
+     *
+     * @see this.selectClass for parameters
+     */
+    parallax: function (previewMode, widgetValue, params) {
+        const isParallax = widgetValue === 'true';
+        this.$target.toggleClass('parallax', isParallax);
+        if (isParallax) {
+            this._initParallax();
+        } else {
+            this._removeParallax();
+        }
+        this.isParallax = isParallax;
+    },
+
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+    /**
+     * @override
+     */
+    _computeVisibility: function () {
+        let $el = this.$target.find('> .s_parallax_bg');
+        $el = $el[0] ? $el : this.$target;
+        const hasBackgroundImage = $el.css('background-image') !== 'none';
+        if (!hasBackgroundImage) {
+            this._removeParallax();
+            this.$target.removeClass('oe_img_bg');
+        }
+        const showParallaxOptions = hasBackgroundImage && !this.$target.hasClass('o_background_video');
+        return this._super(...arguments) && showParallaxOptions;
+    },
```

Comment:
@bvr-odoo The role of `_computeVisibility` is to check the visibility and nothing else. You cannot change the DOM in there. And in fact, why do you need `_computeVisibility` for this? A simple data-dependencies should work here, and if it does not it should be fixed so that it does.